### ISSUE TITLE
Does the app support gateway functionality for ThingSet data to cloud?

### ISSUE THREAD

**Issue by ananya-wattsnext** [ThingSet/thingset-app#1]
Does the application support functionality as a gateway so that we can send ThingSet data to a cloud server?
It would be helpful to know if there are built-in mechanisms to achieve this.

**Comment by martinjaeger** [comment]
No, that's not possible at the moment, even though it should not be very hard to implement.

However, I'm wondering if that would be very useful. A phone is usually not continuously connected to a device. Often apps are even put into sleeping state as soon as the screen is locked. So you would only get very few data forwarded to the cloud.

In other projects we are using a gateway application written in Rust, which can be deployed on a Raspberry Pi for example. It can forward the data from CAN or serial to a WebSocket or MQTT (or directly feed it to a database like VictoriaMetrics so that it can be monitored in Grafana). That application is currently not Open Source, but you can get in contact with me if you are interested in using it.
